src/arch/xtensa/Kconfig

```diff
@@ -8,4 +8,23 @@ config SMP
 	help
 	  Build SMP instead of UP
 
+config CORE_COUNT
+	int "Number of cores"
+	range 1 4
+	depends on CAVS
+	help
+	  Number of cores
+
+config HP_MEMORY_BANKS
+	int "HP memory  baks count"
+	depends on CAVS
+	help
+	  Avalible memory banks cont for HP memory
```

Comment:
Can we say High Performance in the help section instead of HP

Reply:
typo: available.